src/lib/logger/lib/transport.ts

```diff
@@ -0,0 +1,28 @@
+import { ELoglevel, ETransportType } from "../utils/enums";
+
+
+export class BaseTransport {
+    loglvl: ELoglevel = ELoglevel.WARN;
+    showDate: boolean = false;
+    showBaseComment: boolean = false;
+    showLoglevel: boolean = true;
+    baseComment: string = `Default`;
+
+    constructor(baseTransport?: BaseTransport) {
+        if (baseTransport) {
+            this.baseComment = baseTransport.baseComment;
+            this.loglvl = baseTransport.loglvl;
+            this.showBaseComment = baseTransport.showBaseComment;
+            this.showDate = baseTransport.showDate;
+            this.showLoglevel = baseTransport.showLoglevel;
+        }
+    }
+}
+
+export class DefaultConsolTransport extends BaseTransport {
+    type: ETransportType = ETransportType.console;
+
+    constructor(baseTransport?: BaseTransport) {
```

Comment:
You could remove that constructor, since he does nothing just call the parent ;)

Reply:
fixed with https://github.com/senseexcel/ser-ext-ondemand/commit/20834c08ae836aeaa95d2c8de8465dacbb87c2df